### ISSUE TITLE
Are some music examples missing in Weitzmann 1860?

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#47]
I wonder if some musical examples are missing in the scan:

![image](https://user-images.githubusercontent.com/75686579/108996625-e7c09380-769e-11eb-8ec5-f8237b85c4f0.png)
There is a weird space left on top of the page. The previous page had some musical examples numerated with 1., 2., 3. Here we see a little 4. on the left side, as if it would continue the examples, but there is none.

It reminded me of an earlier page:

![image](https://user-images.githubusercontent.com/75686579/108997164-92d14d00-769f-11eb-9c79-ef2cba665c6b.png)

Here also, we find an empty space, and even the caption on the top right, which is usually included in this book whenever music is taken from works.

**Comment by m-k94** [comment]
On page 66, it also appears to be text that's missing. It could be a music example followed by a paragraph, as the cut off line above the bottom-example suggests:

![image](https://user-images.githubusercontent.com/75686579/112723298-ff916e80-8f0d-11eb-82ee-95e41c46b2de.png)

**Comment by fabianmoss** [comment]
Yes apparently there are missing bits in the scan. That's very unfortunate but these things only turn up during a closer look at the sources. In all the examples you mention above, it is pretty obvious that there is something missing in the scan but other instances might go undetected. My suggestion is that you add a new element type "Missing". At a later stage we might be able to get the information from other sources. But for now, I think just marking them up as missing is a good way to not lose track of them while you still can continue with the transcription. Please add this to the Guidelines, too.

**Comment by m-k94** [comment]
Done. 

I added it as a structure type called "missing" and used it in conjunction with "textRegion". (as far as I know it's not possible to create element types as you have suggested). I also added a note about it in the guidelines.

I have identified missing regions on the following pages: 16?, 20, 28, 37?, 43?, 54?, 56, 62, 64, 66, 68, 69, 76.

The ones with the question mark might be complete, but have some slightly weird spacing that suggests the possibility. For the most part, it seems like music examples are missing.

I am leaving this issue open until we find a way to replace them. 

Also, keep in mind that some of the regions structureTypes might change between "-continued" or not depending on what is missing between them.

**Comment by m-k94** [comment]
This issue appears to apply to Oettingen as well:

![image](https://user-images.githubusercontent.com/75686579/118356942-d5f0dd00-b577-11eb-871b-122821c00f53.png)
